src/pages/Game/index.tsx

```diff
@@ -1,5 +1,34 @@
-const GameMainPage = () => {
-  return <h1>GameMainPage</h1>
+import { Button } from '@/components'
+import { FC } from 'react'
+import * as S from './Game.styles'
+import { Modal } from './components/Modal/index'
+import * as C from './constants/texts'
+
+const GameMainPage: FC = () => {
+  return (
+    <S.DividedArea>
+      <S.HalfBox align="left">
+        <S.ThumbnailImg src="assets/img/FlagMoon.svg" />
+        <Modal align="left"></Modal>
```

Comment:
태그 사이에 아무런 요소가 없다면 단일 태그로 닫으셔도 됩니다~!

```suggestion
        <Modal align="left" />
```

Reply:
전에도 지적 한번 주셨죠 ㅋㅋ 알고는 있는데, 자동완성이 되버려서 자꾸 놓치네요., 잘 살펴볼께요~